### ISSUE TITLE
rhythmbox 3.3.1 freezes

### ISSUE THREAD

**Issue by David-UPCT** [fossfreedom/coverart-browser#384]
I use archlinux, and coverart-plugin 2.1 works with rhythmbox 3.3 in gnome 3.20. Today archlinux has updated rhythmbox to 3.3.1, and rhythmbox freezes unless I disable coverart plugin.

This is what I get in RB 3.3 (RB works): http://pastebin.com/5ekq5sep

This is what I get in RB 3.3.1 with coverart enabled (RB freezes at start): http://pastebin.com/7UyGeG3L

When I start RB 3.3.1 with coverart disabled, everything seems fine: http://pastebin.com/Wt6Vsi2L , but when I enable coverart RB inmediately hungs: http://pastebin.com/HQqJYhUq

I'm afraid I don't understand very well the warnings and error messages I've pasted, I don't know if you can extract some information from them. I wish you could solve the problem, IMO rhythmbox is useless without you extension.

**Comment by fossfreedom** [comment]
yeah - sorry about that.  RB 3.3.1 has dropped support for webkit v1 which the plugin still uses.

The plugin needs to be migrated to webkit v2.

In the interim I've got a branch (gtk318) which disables all the code that requires webkit - however it still needs more work because GTK 3.18 & GTK3.20 has broken the size-allocation of various panels that are displayed.

I'm working full time on another project at the moment but will hopefully get sometime next month to revisit coverart

**Comment by David-UPCT** [comment]
I'm not sure if this issue should be close, but coverart seems to work in Archlinux with the recent rhythmbox 3.4: http://pastebin.com/SYCcU3Bm (ignore the llyrics messages, they are related with a different plugin which unfortunately doesn't work in RB 3.4)

**Comment by DragoonZ** [comment]
@David-UPCT which version of coverart are u using? I still cannot get it work.
archlinux and rhythmbox 3.4 too

**Comment by David-UPCT** [comment]
rhythmbox-plugin-coverart-browser 2.1-1 and rhythmbox-plugin-coverart-search-providers 1.2-1. I keep a security copy of both compiled packages:
[rhythmbox coverart (archlinux packages).zip](https://github.com/fossfreedom/coverart-browser/files/442876/rhythmbox.coverart.archlinux.packages.zip)

**Comment by BenBenna** [comment]
yes! it works.

THANK YOU

-- 
mit freundlichen Grüßen

Bernd Romeike
von SchriftBild Romeike in Düsseldorf (SBRD.de)

Marschallstr. 36
D-40477 Düsseldorf
fon +49 211 9385381
fax +49 211 9385384
www.SBRD.de

**Comment by David-UPCT** [comment]
I'm writing only to confirm that rhythmbox coverart still works well in rhythmbox 3.4.1 in archlinux. I don't know what happens in other distros.
